doc/visual-programming/source/widgets/visualize/treeviewer.rst

```diff
@@ -0,0 +1,92 @@
+Tree Viewer
+===========
+
+.. figure:: icons/tree-viewer.png
+
+A visualization of classification and regression trees. 
+
+Signals
+-------
+
+**Inputs**:
+
+-  **Tree**
+
+   Classification or regression tree.
+
+**Outputs**:
+
+-  **Selected Data**
+
+   Data from a selected tree node.
+
+-  **Data**
+
+   Data set with an additional attribute for selection labels.
+
+Description
+-----------
+
+This is a versatile widget with 2-D visualization of a `classification
+tree <https://en.wikipedia.org/wiki/Decision_tree_learning>`_. The user 
+can select a node, instructing the widget to output the
+data associated with the node, thus enabling explorative data analysis.
+
+.. figure:: images/tree-viewer-stamped.png
+
+1. Information on the input.
+
+2. Display options:
+
+   - Zoom in and zoom out
+   - Select the tree width. The nodes display information bubbles when hovering over them.
+   - Select the depth of your tree.
+   - Select edge width. The edges between the nodes in the tree graph are drawn based on the selected edge width.
+      -  All the edges will be of equal width if *Fixed* is chosen.
+      -  When *Relative to root* is selected, the width of the edge will
+         correspond to the proportion of instances in the corresponding
+         node with respect to all the instances in the training data. Under
+         this selection, the edge will get thinner and thinner when
+         traversing toward the bottom of the tree.
+      -  *Relative to parent* makes the edge width correspond to the proportion
+         of instances in the nodes with respect to the instances in their
+         parent node.
+   -  Define the target class, which you can change based on classes in the data. 
+
+3. Press *Save image* to save the created classification tree graph to your computer as a *.svg* or *.png* file. 
+
+4. Produce a report. 
+
+Examples
+--------
+
+Below, is a simple schema, where we have read the data, constructed the
```

Comment:
I suppose so. This was done by @karoema, so it's a professional review. I just moved the file to appropriate place. :)